test/demo/utils/providerRemote.ts

```diff
@@ -0,0 +1,21 @@
+import {Identity} from 'hadouken-js-adapter';
+
+import {Store} from '../../../src/provider/store/Store';
+import {Action} from '../../../src/provider/store/Actions';
+
+import {OFPuppeteerBrowser, BaseWindowContext} from './ofPuppeteer';
+import {serviceIdentity} from './constants';
+
+export interface ProviderContext extends BaseWindowContext {
+    store: Store;
+    targetWindow: Identity;
```

Comment:
Think this wouldn't actually be present on the provider?..

Reply:
my bad. that's a leftover, I was testing something. removed.